staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2314,6 +2314,26 @@ type ResourceRequirements struct {
 	// More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
 	// +optional
 	Requests ResourceList `json:"requests,omitempty" protobuf:"bytes,2,rep,name=requests,casttype=ResourceList,castkey=ResourceName"`
+	// Claims lists the names of resources, defined in spec.resourceClaims,
+	// that are used by this container.
+	//
+	// This is an alpha field and requires enabling the
+	// DynamicResourceAllocation feature gate.
+	//
+	// This field is immutable.
+	//
+	// +listType=set
+	// +featureGate=DynamicResourceAllocation
+	// +optional
+	Claims []ResourceClaim `json:"claims,omitempty" protobuf:"bytes,3,opt,name=claims"`
```

Comment:
We just ran across this while reviewing https://github.com/kubernetes/kubernetes/pull/107776

The `ResourceRequirements` type is also used within PVC objects, and the "clear if feature-gate is disabled" logic is only wired to podspec-handling. That means in 1.26 PVC objects can be persisted with data in this field.

Can you open a PR to clear this field in PVC create/update flows, and file a follow-up issue to consider splitting the type used in PVCs so it doesn't continue inheriting work done that is targeted at pods?